README.md

```diff
@@ -423,54 +437,79 @@ ownership in a company you work for.
+In general, startups rely on investors to help fund rapid growth.
 
 🄳 In order to finance building or scaling their business, startups **fundraise** by selling
 shares in their business to [investors](http://www.investopedia.com/terms/i/investor.asp) in
 exchange for capital.
+(In this way, employees and investors share ownership in private companies.
+When a company [goes public](#going-public), anyone can buy stock and share in the successes
+and failures of a company through partial ownership.)
 For startups that aspire to grow rapidly, it’s likely they will fundraise from investors
 called **venture capitalists**.
 
 A startup goes through [several stages of growth](#stages-of-a-startup) as it fundraises
 based on evidence that it will make money in the future.
 
-- 🄳 As these companies add (or “issue”) stock, the outstanding shares goes up, and the
-  percentage ownership of each shareholder goes down.
-  This is called [**dilution**](http://www.investopedia.com/terms/d/dilution.asp). That’s a
-  confusing word to most humans, but the concept is relatively simple:
-  - Dilution doesn’t necessarily mean that as a shareholder you’re losing anything-usually, as
-    a company raises money and issues stock, the hope and intention is that what you do have
-    is worth more.
-    The size of your slice gets relatively smaller, but if the company is growing, the size of
-    the cake gets bigger.
+- 🄳 As these companies add (or “issue”) stock, the number of outstanding shares goes up,
+  and the percentage ownership of each shareholder goes down.
+  This is called [**dilution**](http://www.investopedia.com/terms/d/dilution.asp).
+  - Dilution doesn’t necessarily mean that you’re losing anything as a shareholder.
+    As a company issues stock and raises money, the smaller percentage of the company you *do*
+    have should be *worth* more.
+    The size of your slice gets relatively smaller, but, if the company is growing, the size
+    of the cake gets bigger.
   - For example, a typical startup might have three rounds of funding, with each round of
     funding issuing 20% more shares.
     At the end of the three rounds, there are more outstanding shares-roughly 70% more in this
     case, since 120%×120%×120% is 173%-and each shareholder owns proportionally less of the
     company.
-- 🄳 The [**valuation**](https://en.wikipedia.org/wiki/Valuation_(finance)) of the company is the
-  present value investors believe the company has.
-  If the company is doing well, making more revenue or gaining users (indicating future
-  revenue), the company’s valuation will usually be going up.
-  That is, the **price** to an investor for buying one share would be increasing.
+- 🄳 The [**valuation**](https://en.wikipedia.org/wiki/Valuation_(finance))
+  of the company is the present value investors believe the company has.
+  If the company is doing well, growing revenue by gaining customers or measuring high
```

Comment:
"measuring high anticipation" sounds strange and unclear to my ear. Let's go with something like:

"growing revenue, gaining customers, or in other ways building evidence that there is a large market for their product in the future"